pages/summary.js

```diff
@@ -33,7 +33,15 @@ export default function Summary({ data, queryString }) {
       <RegulomeVersionTag />
       <PagePreamble />
       <DataPanel>
-        <DataAreaTitle>This search has found {total} variant(s).</DataAreaTitle>
+        <DataAreaTitle>
+          This search has found <b>{total}</b> variant(s).{" "}
+          {total > variants.length ? (
+            <span>
+              {" "}
+              Only <b>{variants.length}</b> are shown.
+            </span>
+          ) : null}
```

Comment:
You could simplify this to` {total > variants.length && (<> Only …</>)}` since `false` and `undefined` also render nothing. It makes very little or no functional difference though.

Reply:
Thanks for the tips!